tests/python/shared/utils/dump_objects.py

```diff
@@ -34,41 +36,39 @@ def clean_list_response(data: dict[str, Any]) -> dict[str, Any]:
     return data
 
 
-if __name__ == "__main__":
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--assets-dir", type=Path, default=ASSETS_DIR)
+    args = parser.parse_args()
+
+    assets_dir: Path = args.assets_dir
+
     annotations = {}
 
-    for obj in [
-        "user",
-        "project",
-        "task",
-        "job",
-        "organization",
-        "membership",
-        "invitation",
-        "cloudstorage",
-        "comment",
-        "issue",
-        "webhook",
-        "label",
-        "quality/report",
-        "quality/conflict",
-        "quality/setting",
-        "consensus/setting",
-    ]:
-        response = get_method("admin1", f"{obj}s", page_size="all")
-
-        filename = f"{obj}s.json".replace("/", "_")
-        with open(ASSETS_DIR / filename, "w") as f:
-            f.write(json.dumps(clean_list_response(response.json()), indent=2, sort_keys=True))
-
-        if obj in ["job", "task"]:
+    for dump_path in assets_dir.glob("*.json"):
+        endpoint = dump_path.stem.replace("_", "/")
+
+        if endpoint == "annotations":
+            continue  # this will be handled at the end
+
+        response = get_method("admin1", endpoint, page_size="all")
+
+        with open(dump_path, "w") as f:
+            json.dump(clean_list_response(response.json()), f, indent=2, sort_keys=True)
+
+        if endpoint in ["jobs", "tasks"]:
+            obj = endpoint.removesuffix("s")
```

Comment:
It looks like it's better to pass some config file instead. Something like a `{ url: filename }` dict.

Reply:
I don't think I understand your proposal. Could you elaborate?

Reply:
I the point is to extend the usability of the script, it's better to provide a way to use it with some user-passed configuration instead of hardcoding behavior for specific endpoints.

Reply:
I don't think so. The less configuration is needed, the more usable it is.

You could say that the set of JSON dump files in the asset directory is the "configuration".

Reply:
Ok, so if the script depends on its outputs, how a new endpoint should be added?

Reply:
So, what do you think?

Reply:
As I said, it's better to support configuration files instead. I'm not sure I've seen the current design anywhere else. The most similar thing that comes to my mind is drop-in files in /etc, but in that case most of them are either configuration files, scripts, or files with a specific format. If you want to keep the current design, at least add documentation - in the script and, probably, in `README.md` of python tests.

Reply:
> As I said, it's better to support configuration files instead.

But why is it better?

> I'm not sure I've seen the current design anywhere else. The most similar thing that comes to my mind is drop-in files in /etc, but in that case most of them are either configuration files, scripts, or files with a specific format.

It's an ad-hoc script, so I'm not too concerned about using ad-hoc techniques. 🤷

> If you want to keep the current design, at least add documentation

Sure, added.